### DISCUSSION TITLE
ability to orderBynearest neighbor on eloquent models

### DISCUSSION THREAD

**Discussion by vsamadian** [karam-mustafa/laravel-geographical-calculator#19] [Q&A] [answered]
is it possible to sort models that have Latitude and Longitude fields , by function like orderByNearest base on single point ?

**Comment by karam-mustafa** [comment, marked as answer]
Unfortunately at the moment, the package deals only with coordinate points, and yes I am thinking of adding features to deal with coordinates in eloquent 
Thanks
